config.py

```diff
@@ -891,3 +891,6 @@ def create_transaction(db):
 
     FEATURE_ASSIGN_OAUTH_TOKEN = True
     DEFAULT_NAMESPACE_AUTOPRUNE_POLICY: Optional[Dict[str, str]] = None
+
+    # Allows users to set up notifications on image expiry, can remove flag once feature is tested
+    FEATURE_IMAGE_EXPIRY_TRIGGER = False
```

Comment:
you can default this to True, we can set it to false manually

Reply:
Since, this is a temporary flag, I thought why add it in app-interface and clean it afterwards if that makes sense. I plan to remove this flag after QE is done testing.

Reply:
yeah that's fair, i usually like to have the control to enable/disable certain functionality it some cases. We can leave it